release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -3005,3 +3005,24 @@ link:https://access.redhat.com/solutions/6968056[{product-title} 4.9.43 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-45"]
+=== RHSA-2022:5879 - {product-title} 4.9.45 bug fix update and security update
+
+Issued: 2022-08-09
+
+{product-title} release 4.9.45, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5879[RHSA-2022:5879] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5878[RHBA-2022:5878] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6970809[{product-title} 4.9.45 container image list]
+
+[id="ocp-4-9-45-bug-fixes"]
+==== Bug fixes
+
+* Previously, the metrics endpoints for Cinder CSI Driver and Cluster CSI Driver used unsecured Transport Layer Security (TLS) configurations. This allowed attackers access to vulnerable cryptography where they can decrypt and modify the endpoints. With this update, the TLS configurations are more secure and eliminate access to vulnerable cryptography. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110255#[*BZ#2110255*])
```

Comment:
"This allowed attackers access" - to make clearer, I suggest "These configurations allowed..." or something to indicate what "this" is referring to.

Reply:
Done! :) Thanks for your help!

Reply:
> access to vulnerable cryptography where they can decrypt and modify the endpoints

I'm not sure this is right. The original docs text says the attacker can decrypt or modify the **traffic** to and from these endpoints, not the endpoints themselves.

Reply:
@mburke5678 You are def. right! Thanks for catching that! Updates have been made. :)